data/config.yaml

```diff
@@ -1,8 +1,8 @@
 _id: d779730c7cc94155877a24e2e8be2ad1
-created: '2022-04-19T11:53:46.882828'
+created: '2022-02-09T14:59:17.579871'
```

Comment:
I am seeing this file change in each commit still. Firstly, merging the latest main branch should fix the issue of the tests changing this all the time. Ideally we would also remove this file from the PR, but I don't think it hurts anything to leave it in and working out how to undo it might be more effort than it is worth,

Reply:
If you're just reviewing changes since you last reviewed, that might be the reason. Note that the timestamp here is _older_ than the one it replaced. I don't see it in the overall diff.

Reply:
Ah, that is probably the reason.